### ISSUE TITLE
[plugin-graphql] Missing `graphql` dependency

### ISSUE THREAD

**Issue by fabiodrg** [rollup/plugins#1030]
- Rollup Plugin Name: plugin-graphql
- Rollup Plugin Version: 1.0.0
- Rollup Version: 2.58.3
- Operating System (or Browser): Fedora 34 (Linux)
- Node Version: 14.18.1
- Link to reproduction _(⚠️ read below)_: https://replit.com/join/qeldlfsiwr-fabiodrg

### Expected Behavior

The `@rollup/plugin-graphql` has a dependency on `graphql-tag` (https://github.com/apollographql/graphql-tag), which in turn depends on `graphql` (https://github.com/graphql/graphql-js). As stated in `graphql-tag` README:

> graphql-tag uses the reference graphql library under the hood as a peer dependency, so in addition to installing this module, you'll also have to install graphql.

In other words, when `graphql-tag` is installed as dependency for `@rollup/plugin-graphql`, npm wont install `graphql`. Therefore, I suggest setting `graphql` as dependency for `@rollup/plugin-graphql` so that the plugin works out of the box.

### Actual Behavior

Because `graphql` dependency is missing, when the plugin is loaded, it crashes!

**Comment by shellscape** [comment]
Thanks for the issue. Unfortunately it's not a valid one. We already have that set as a `peerDependency`, see: https://github.com/rollup/plugins/blob/6eb661692f5b8d8fc4e3b61ff748f49fab1e82ec/packages/graphql/package.json#L47-L50

It looks like you might be starting out in the ecosystem, so I'd like to point you to the NPM docs on dependencies: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies. All of the package managers will output peerDependency warnings when installing, and NPM v7 installs peerDependencies automatically. It's very important to understand how dependency management works within Node and NPM, and how they're configured in package.json.

**Comment by fabiodrg** [comment]
Thanks for the clarification and resources. I am on NPM v6.x and NPM shows a warning, but I missed it...!
Have a nice day!
